src/chttpd/src/chttpd.erl

```diff
@@ -159,7 +167,7 @@ handle_request(MochiReq0) ->
 
 handle_request_int(MochiReq) ->
     Begin = os:timestamp(),
-    case config:get("chttpd", "socket_options") of
+    case config:get("chttpd", "socket_options", ?DEFAULT_SOCKET_OPTIONS) of
```

Comment:
When you use `config:get/3` you cannot get `undefined` atom. I believe it should be written as:
```
SocketOptsCfg = config:get("chttpd", "socket_options", ?DEFAULT_SOCKET_OPTIONS),
{ok, SocketOpts} = couch_util:parse_term(SocketOptsCfg),
ok = mochiweb_socket:setopts(MochiReq:get(socket), SocketOpts)
```

Reply:
Changed, Thanks for the review @iilyak